### PR TITLE
rust-dev 20230713

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,59 +1,58 @@
...
 
-
 source:
   # Note! This source file specification is structured specifically to interact
   # well with the autotick bot. Importantly, the `patches:` block actually
   # applies to *all* of these inputs, not just the last one in the list, because
   # of the way that the YAML selector processing works.
-  - url: https://static.rust-lang.org/dist/{{ year }}-{{ month }}-{{ day }}/rust-nightly-x86_64-unknown-linux-gnu.tar.gz  # [linux and x86_64]
-    sha256: f6aabe9b6f2ff1ec656317f004feaf9315b55a5234eddaefa9a72d44c5a1aaa6  # [linux and x86_64]
+    - url: https://static.rust-lang.org/dist/{{ year }}-{{ month }}-{{ day }}/rust-nightly-x86_64-unknown-linux-gnu.tar.gz  # [linux and x86_64]
```
Comment (pkgw) [inline recipe/meta.yaml:15]: Indentation problem here, it looks like.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,59 +1,58 @@
...
-    sha256: 210a80f2563a1b2c2de5bad071d1935e08346c59c056b8a5c209655e140b7f21  # [linux and x86_64]
+    sha256: 54abf1b60baf6d129691c901e0d2cb3c123b83f758d400e4fe8f9b9d34a9fbe3  # [linux and x86_64]
     folder: rust-std  # [linux and x86_64]
   - url: https://static.rust-lang.org/dist/{{ year }}-{{ month }}-{{ day }}/rust-std-nightly-i686-linux-android.tar.gz  # [linux and x86_64]
-    sha256: fec6978bc2fb8da95d86e58a9eda416aac076356b9525c632db199d445dc7bb7  # [linux and x86_64]
+    sha256: db007f26e7fc5ca682829b0e1f79c08bd501b23739e17902f10fcfb1261f8b81  # [linux and x86_64]
     folder: rust-std  # [linux and x86_64]
-  - url: https://static.rust-lang.org/dist/{{ year }}-{{ month }}-{{ day }}/rust-std-nightly-x86_64-linux-android.tar.gz  # [linux and x86_64]
-    sha256: faca896cf91e608342ca6e539b2d90d066e9b4b0bf14b316b61dc92914d2fa93  # [linux and x86_64]
+  - url: https://static.rust-lang.org/dist/{{ year }}-{{ month }}-{{ day }}/rust-std-nigthly-x86_64-linux-android.tar.gz  # [linux and x86_64]
```
Comment (pkgw) [inline recipe/meta.yaml:54]: Typo here.

**Review by pkgw — CHANGES_REQUESTED** [review]
I think there are just a couple of typos here.

**Comment by xhochy** [comment]
Merging since I adressed all comments.

**Pull request merged**